src/snowflake/snowpark/modin/config/envvars.py

```diff
@@ -870,6 +870,16 @@ class DaskThreadsPerWorker(EnvironmentVariable, type=int):  # pragma: no cover
     default = 1
 
 
+class SnowflakePandasTransferThreshold(EnvironmentVariable, type=int):
+    """
+    Targeted max number of dataframe rows which should be transferred from
+    Snowflake when using hybrid execution.
+    """
+
+    varname = "SNOWFLAKE_PANDAS_MAX_XFER_ROWS"
+    default = 100_000
+
+
```

Comment:
Unfortunately these environment variables remain from when Snowpark pandas was a fork of modin. You can monkey patch this class onto `modin.config`:
```suggestion
modin.config.SnowflakePandasTransferThreshold = SnowflakePandasTransferThreshold
```

Reply:
blah. OK.